core/src/apps/cardano/layout.py

```diff
@@ -394,6 +390,55 @@ async def confirm_withdrawal(
     await require_confirm(ctx, page1)
 
 
+async def confirm_catalyst_registration(
+    ctx: wire.Context,
+    public_key: str,
+    staking_path: List[int],
+    reward_address: str,
+    nonce: int,
+) -> None:
+    pages: List[ui.Component] = []
+
+    page1 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+    page1.bold("Confirm voting key")
+    page1.bold("registration.")
```

Comment:
I'd drop the dot at the end - seems kind of redundant to me, unless it's some wide-spread UI-pattern, but I doubt so

Reply:
Also the "Confirm" verb seems redundant to me and not consistent e.g. with the pool registration flow where we just say "Pool registration <etc>" in the initial prompt. So I'd just print here `Catalyst voting key registration`

Reply:
https://github.com/vacuumlabs/trezor-firmware/pull/57/commits/f73f563cd7b142506f75b45773fe521898ae04ab